components/board/genericlinux/config.go

```diff
@@ -11,25 +11,13 @@ import (
 
 // A Config describes the configuration of a board and all of its connected parts.
 type Config struct {
-	I2Cs              []board.I2CConfig                   `json:"i2cs,omitempty"`
-	SPIs              []board.SPIConfig                   `json:"spis,omitempty"`
 	AnalogReaders     []mcp3008helper.MCP3008AnalogConfig `json:"analogs,omitempty"`
 	DigitalInterrupts []board.DigitalInterruptConfig      `json:"digital_interrupts,omitempty"`
 	Attributes        utils.AttributeMap                  `json:"attributes,omitempty"`
```

Comment:
if attributes is unused, we may as well remove it too.

Reply:
Okay, this is fixed in https://github.com/viamrobotics/rdk/pull/3264